crates/cxx-qt-lib/build.rs

```diff
@@ -270,6 +270,7 @@ fn main() {
     builder.flag_if_supported("/permissive-");
     // GCC + Clang
     builder.flag_if_supported("-std=c++17");
+    builder.flag_if_supported("-Wa,-mbig-obj");
```

Comment:
could you add a comment above this line in both cases stating that `MinGW requires big-obj otherwise debug builds fail` or similar

Reply:
Done